### ISSUE TITLE
JavaScript best practices - Eva

### ISSUE THREAD

**Issue by BucurEva87** [WalterOkumu/to-do-list#6]
Found no issue regarding the best practices of javascript coding, but a bug. In order to replicate this behavior add at least one task, mark it as completed then press the "clear all completed" button. Once this is done try adding a new task and it won't work (an error will be thrown in your console).

### Suggestion
 - when you're adding a new task you're giving it an index of either 1 (if that's the first task you're adding) or the last task's in the array index plus one. Please remember that the tasks could be in any order inside the array, so the last task isn't guaranteed to have the highest index. Kindly search for the task with the max index and the give your new task an index of the max index you found plus one.

**Comment by WalterOkumu** [comment]
Thanks, I've since fixed this issue
